pytest_plugins/upgrade/scenario_workers.py

```diff
@@ -58,11 +56,7 @@ def set_post_upgrade_hostname(request, shared_workers):
             pre_hostname = get_worker_hostname_from_testname(pre_test_name, shared_workers)
             if (pre_hostname is None) or (pre_hostname not in settings.server.hostnames):
                 pytest.skip(
-                    'Skipping the post_upgrade test as the pre_upgrade hostname is not found!'
+                    'Skipping the post_upgrade test as the pre_upgrade hostname was not found!'
                 )
-            # Read the worker id of pre test name
-            settings.server.hostname = env.host_string = pre_hostname
-            env.user = 'root'
             configure_nailgun()
             configure_airgun()
-            set_hammer_config()
```

Comment:
What's an alternative to this? how we're setting up a hammer config then and don't we need these requirements anymore?
(sorry, I might be missing some discussion and missed asking this in earlier PR)

Reply:
@Gauravtalreja1 Sorry for the late response, I missed it and accidentally discovered it. :sweat_smile: 

The `set_hammer_config` method invocation that you see here is setting the hammer credentials to the Fabric env.
https://github.com/SatelliteQE/automation-tools/blob/de090390670a0c9cdb5148137a88d0ace00116de/automation_tools/satellite6/hammer.py#L64-L68
As we do not use Fabric anymore, we are safe to remove the method invocation and therefore remove automation-tools from our requirements.